packages/admin-ui/ui/src/domain/orders/edit/variants-table.tsx

```diff
@@ -46,6 +49,58 @@ const VariantsTable: React.FC<Props> = (props) => {
     }
   }, [count])
 
+  const VariantInventoryCell = ({ row: { original } }) => {
+    const { getLocationNameById } = useStockLocations()
+
+    const { variant, isLoading } = useAdminVariantsInventory(original.id)
+
+    if (isLoading) {
+      return (
+        <div className="flex justify-end">
+          <Skeleton isLoading={true}>
+            <div className="h-[20px] w-[50px]" />
+          </Skeleton>
+        </div>
+      )
+    }
+
+    if (!isLoading && (!variant || !variant.inventory)) {
+      return <div className="text-right">{original.inventory_quantity}</div>
+    }
+
+    const { inventory } = variant
+
+    const total = inventory[0].location_levels.reduce(
+      (sum: number, location_level: InventoryLevelDTO) =>
+        (sum += location_level.stocked_quantity),
+      0
+    )
+
+    const LocationTooltip = (
+      <>
+        {inventory[0].location_levels.map(
+          (location_level: InventoryLevelDTO) => (
+            <div key={location_level.id} className="font-normal">
+              <span className="font-semibold">
+                {location_level.stocked_quantity}
+              </span>
+              {" in "}
+              {getLocationNameById(location_level.location_id)}
+            </div>
+          )
+        )}
+      </>
+    )
+
+    return (
+      <Tooltip content={LocationTooltip} side="top" className="translate-x-1/4">
+        <div className="text-right">
+          {total} in {inventory[0].location_levels.length} location(s)
```

Comment:
suggestion: add this https://www.npmjs.com/package/pluralize and remove the `(s)`